### ISSUE TITLE
sharex

### ISSUE THREAD

**Issue by rafalohaki** [ZizzyDizzyMC/linx-server#7]
is there sharex config?

**Comment by ZizzyDizzyMC** [comment]
Yes, there is a sharex config here: https://github.com/ZizzyDizzyMC/linx-server/blob/master/put.icu.sxcu 
While this configuration is unique to Put.icu it is easily adapted to your own domain name.
